rails_docker/Gemfile.txt

```diff
@@ -20,7 +20,6 @@ gem 'rails_admin'
 
 # Authentications & Authorizations
 gem 'devise' # Authentication solution for Rails with Warden
-gem 'doorkeeper' # OAuth 2 provider
 gem 'pundit' # Minimal authorization through OO design and pure Ruby classes
 
 source 'https://rails-assets.org' do
```

Comment:
Can we have another PR to remove this? Don't think we are going to `https://rails-assets.org` with Webpack